src/read/unit.rs

```diff
@@ -4207,11 +4207,7 @@ mod tests {
 
     #[test]
     fn test_attribute_udata_sdata_value() {
-        let tests: &[(
-            AttributeValue<EndianSlice<LittleEndian>>,
-            Option<u64>,
-            Option<i64>,
-        )] = &[
+        let tests: &[(AttributeValue<EndianSlice<LittleEndian>>, _, _)] = &[
```

Comment:
I think this is better left how it is. I assume this is `clippy::type_complexity`, but hiding complexity doesn't make it better.